src/containers/RegisterContainer.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { connect } from 'react-redux'
+import RegisterComponent from '../components/RegisterComponent'
+import { fetchData } from '../actions/index';
+class RegisterContainer extends React.Component{
+    constructor(props){
+        super(props);
+        this.hadleSubmit = this.hadleSubmit.bind(this);
+    }
+    hadleSubmit(e){
+        e.preventDefault();
+        //localStorage.setItem('email',e.target.email.value);
```

Comment:
Why so much commented code. Please remove all console.log. Use some linting tools also. and Please take care of Indentations